MAP_ANONYMOUS is missing on Mac

On my Mac, I get this:

```
error[E0425]: cannot find value `MAP_ANONYMOUS` in module `libc`
  --> src/us/mod.rs:34:38
   |
34 |             libc::MAP_SHARED | libc::MAP_ANONYMOUS,
   |                                      ^^^^^^^^^^^^^ not found in `libc`

```

Now here's what's a little odd... a Google search reveals lots of people (using C) talking about MAP_ANONYMOUS being absent on Macs, which I guess is why it's not in `libc`. But it's definitely present on my system. This C program compiles:

```
#include<stdio.h>
#include<sys/mman.h>
int main() {
  printf("%d", MAP_ANONYMOUS);
}
```

--- a/src/unix/bsd/apple/mod.rs
+++ b/src/unix/bsd/apple/mod.rs
@@ -772,6 +772,7 @@ pub const MAP_SHARED: ::c_int = 0x0001;
 pub const MAP_PRIVATE: ::c_int = 0x0002;
 pub const MAP_FIXED: ::c_int = 0x0010;
 pub const MAP_ANON: ::c_int = 0x1000;
+pub const MAP_ANONYMOUS: ::c_int = MAP_ANON;
 
 pub const VM_FLAGS_FIXED: ::c_int = 0x0000;
 pub const VM_FLAGS_ANYWHERE: ::c_int = 0x0001;

